docs/modules/ROOT/pages/configuration-store.adoc

```diff
@@ -0,0 +1,172 @@
+mounted to the configuration store kubernetes resource. It is assumed a PV will meet most projects needs and is the most widely accepted 
+structured solution. The configuration service will then consume these properties files and place them into the configuration store. Please 
+see the configuration-store https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-configuration-store-chart/README.md#persistent-volume-properties[helm chart documentation,role=external,window=_blank] for more information on mounting the PV to the service.
+
+===== Properties File Locations
+Within the configuration store's helm chart, one can specify environment variables with the URIs that specify where the project's various 
+properties files are located within the PV. It is required to specify a base property URI that houses the base/default property 
+configurations. In addition to the base URI, one can optionally specify a secondary URI for properties files that house the environment
+specific configurations. 
+
+NOTE: Multiple configuration properties files can be stored at either URI.
+
+Since it is common practice to define separate Helm chart values for each environment of a project's development lifecycle, it is 
+encouraged to define one shared base URI and respective environment specific URIs, each housing the relevant overrides.
+
+The following Helm chart configuration demonstrates the URI specifications for an example CI deployment:
+[source,yaml]
+----
+  env:
+    BASE_PROPERTY: <URI housing base property configurations>
+    ENVIRONMENT_PROPERTY: <URI housing CI-specific property overrides/augmentations>
+----
+
+==== Backend Storage Options
+The universal configuration store offers robust production ready configuration storage using https://developer.hashicorp.com/vault[Vault,role=external,window=_blank]. This is an excellent solution for large 
+enterprise applications. For smaller applications looking for a more light weight but still robust storage option, the configuration store 
+can be set to use https://github.com/TechnologyBrewery/krausening[Krausening,role=external,window=_blank]. The configuration store can also be set up to use in memory storage, this is best
+for quick local testing.
+
+The default property storage is Krausening. To change it to use one of the other options, set the environment
+variable `STORAGE_CLASS` to the desired value. The values currently available are `krausening`, `vault`, and `inMemory`.
+
+=== Usage
+After the configurations are ingested and reconciled within the store, the next step is to access them within a project using one of 
+the two available methods: the API endpoint or kubernetes resource injection. See the applicable sections below for more details on 
```

Comment:
A: Let's make sure we've capitalized proper nouns like _"Kubernetes"_ throughout the page.